src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -371,4 +381,22 @@ export class ExposureNotificationService {
 
     return finalize({}, lastCheckedPeriod);
   }
+
+  private async processPendingExposureSummary() {
+    const summary = await this.exposureNotification.getPendingExposureSummary();
+    const exposureStatus = this.exposureStatus.get();
+    if (exposureStatus.type === 'diagnosed' || (summary?.matchedKeyCount || 0) <= 0) {
+      return;
+    }
+    const today = new Date();
+    this.exposureStatus.append({
+      type: 'exposed',
+      summary,
+      lastChecked: {
+        timestamp: today.getTime(),
```

Comment:
I assume the broadcast receiver from the framework won't have more than day of delay. So, when we receive data from broadcast receiver, we could assume lastChecked is today.